### PR TITLE
Self-service stats announcement unpublishing

### PR DESCRIPTION
**Pull request by jamiecobbett**
Statistics Announcements have no workflow, so are frequently published when they shouldn't have been.

This adds a very primitive `publishing_state` so that they can be either `published` or `unpublished`.

### CODE REVIEW

File: lib/whitehall/authority/rules/statistics_announcement_rules.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Whitehall::Authority::Rules
+  class StatisticsAnnouncementRules < Struct.new(:actor, :subject)
```
Comment (boffbowsh) [inline lib/whitehall/authority/rules/statistics_announcement_rules.rb]:
```ruby
StatisticsAnnouncementRules = Struct.new(:actor, :subject) do
```

**Comment by boffbowsh** [comment]
Couple of comments, all looks good otherwise though.

**Comment by jamiecobbett** [comment]
Rebased to fix the Struct Rubocop violation.

**Comment by boffbowsh** [comment]
:+1:

**Pull request merged**